src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      name="bigSearchBar"
+      class="searchBar"
+      data-qa="big"
+      >
```

Comment:
```suggestion
    <form
      action="#"
      method="get"
      name="bigSearchBar"
      class="searchBar"
      data-qa="big"
    >
```

закриваюча дужка тегу повинна бути під відкриваючою
<img width="197" alt="image" src="https://user-images.githubusercontent.com/102413088/184664599-4a55dfb8-a384-4c0f-84ae-0db32bd8dcba.png">